src/gateway/WebSocketClient.ts

```diff
@@ -311,7 +308,7 @@ export default class WebSocketClient<
   }
 
   /**
-   * Returns the shard information
+   * Returns shard info for the bot
    */
   async getShardInfo(): Promise<types.ShardInfo> {
     const data = this.options.shardCount === 'auto' ?

```

Comment:
Is that `?` supposed to be there and I'm missing some functionality I didn't know TS had or is it a rogue `?`?

Reply:
That's just a ternary spread over multiple lines

Reply:
Basically what Donovan said here, just a ternary to make code slightly more readable